cmd/move_config.go

```diff
@@ -55,6 +55,9 @@ func runMove(cmd *cobra.Command, args []string) error {
 	return Move(settings.DryRun)
 }
 
+var err error
+var doCleanup bool
```

Comment:
These should not be global. How about creating them just before they are used in the function?

Reply:
done also thanks